config/locales/en.yml

```diff
@@ -209,12 +212,13 @@ en:
         period: Period
         period_end: End date
         period_start: Start date
-      assignment_journal:
+      feedback:
         achievements: What could have been achieved during the last months?
         conversation: I would like to discuss with my supervisor.
         future: Should the assignment continue? If yes, with which goals?
         goals: Which were the most important goals during the last months?
         last: Last assignment journal entry
+        feedbackable_id_and_type: Activities
```

Comment:
Do we need `feedback_activity` here as well? (added in `de`)

Reply:
AFAIR I've just given this a different name so it won't be misleading. I chose some name in de.yml, but I' don't remember what exactly right now.